Tests/GameCoreTests/CubeViewTests.swift

```diff
@@ -0,0 +1,548 @@
+import ComposableArchitecture
+import CubeCore
+import SharedModels
+import SnapshotTesting
+import XCTest
+
+class CubeViewTests: XCTestCase {
```

Comment:
Was going to make `CubeCoreTests` but constructing `CubeSceneView.ViewState.ViewPuzzle` is only convenient from `GameCore` right now...could prob clean up in the future.